### PR TITLE
removing fields from view, moving title to top #157

### PR DESCRIPTION
**Pull request by surreal8**

### CODE REVIEW

**Comment by surreal8** [comment]
@awead could you check my edits in edit_generic_file_spec.rb? just want to make sure my deletions are all good.

File: spec/features/edit_generic_file_spec.rb [hunk 1]
```diff
@@ -18,26 +18,5 @@
...
-      fill_in("generic_file[comments_attributes][2][content]", with: comment2)
-      click_button("upload_submit")
-      expect(find_field("generic_file[comments_attributes][0][content]").value).to eql comment0
-      expect(find_field("generic_file[comments_attributes][1][content]").value).to eql comment1
-      expect(find_field("generic_file[comments_attributes][2][content]").value).to eql comment2
-      within("div.comments-editor") { first(:button, "Remove").click }
-      click_button("upload_submit")
-      expect(find_field("generic_file[comments_attributes][0][content]").value).to eql comment1
-      expect(find_field("generic_file[comments_attributes][1][content]").value).to eql comment2
-    end
```
Comment (awead) [inline spec/features/edit_generic_file_spec.rb:41]: @surreal8 why are we completely removing this test?

**Comment by surreal8** [comment]
@awead I removed many fields from the the asset edit page, this included the comments field. this test was giving me errors because it was not finding that field i'm assuming?

**Comment by awead** [comment]
With this commit, you've removed the ability to add comments to assets. I'm assuming this is only temporary. @scossu ? if so, then we should mark the test pending to indicate this is functionality that will be available again later.

@surreal8 in the original test, insert this line after the `it` statement and leave the rest of the test intact:

``` ruby
it "supports adding and removing" do
  skip "disabled for beta1 release"
```

Otherwise, if the intention is to in fact remove the ability to add comments to asses, then you can remove this entire file since it only consists of this one test.

**Comment by surreal8** [comment]
@awead @scossu ok, i'll add that line now - thanks!

**Comment by scossu** [comment]
@awead @surreal8 That seems a good temporary fix. Test can be re-enabled where appropriate after the beta release.

**Pull request closed without merging**